File: matiasmasso/Mat.NET/DT2/DTOProductChannel.vb
Public Class DTOProductChannel
    Inherits DTOBaseGuid

    Property Product As DTOProduct
    Property DistributionChannel As DTODistributionChannel
    Property Cod As Cods

    Property Inherited As Boolean

    Public Enum Cods
        Inclou
        Exclou
    End Enum

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub
End Class

File: matiasmasso/Mat.NET/DTO/DTOProductChannel.cs
using System;

namespace DTO
{
    public class DTOProductChannel : DTOBaseGuid
    {
        public DTOProduct Product { get; set; }
        public DTODistributionChannel DistributionChannel { get; set; }
        public Cods Cod { get; set; }

        public bool Inherited { get; set; }

        public enum Cods
        {
            Inclou,
            Exclou
        }

        public DTOProductChannel() : base()
        {
        }

        public DTOProductChannel(Guid oGuid) : base(oGuid)
        {
        }
    }
}
